src/OrchardCore/OrchardCore.ResourceManagement.Abstractions/ResourceDebugMode.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace OrchardCore.Settings
+namespace OrchardCore.Settings
```

Comment:
This file should not appear here, try to undo it under Visual Studio or your git client and then push again.